packages/playwright-core/src/server/chromium/videoRecorder.ts

```diff
@@ -137,10 +143,12 @@ export class VideoRecorder {
       for (let i = 0; i < repeatCount; ++i)
         this._frameQueue.push(this._lastFrameBuffer);
       this._lastWritePromise = this._lastWritePromise.then(() => this._sendFrames());
+      this._lastFrameTimestamp += repeatCount / fps;
+    } else {
+      this._lastFrameTimestamp = timestamp - (monotonicTime() - this._launchTimestamp) / 1000;
```

Comment:
This will only increase the duration of the resulting video by the time it took to launch screencast, why do we want this?